src/main/java/com/uid2/admin/job/jobsync/PrivateSiteDataSyncJob.java

```diff
@@ -158,13 +158,14 @@ public void execute() throws Exception {
                 encryptionKeyWriter
         );
         KeyAclSyncJob keyAclSyncJob = new KeyAclSyncJob(keyAclWriter, globalOperators, globalKeyAcls);
+
         siteSyncJob.execute();
         clientSyncJob.execute();
         encryptionKeySyncJob.execute();
         keyAclSyncJob.execute();
         if(config.getBoolean(enableKeysetConfigProp)) {
             Map<Integer, Keyset> globalKeysets = keysetStoreFactory.getGlobalReader().getSnapshot().getAllKeysets();
-            Collection<KeysetKey> globalKeysetKeys = keysetKeyStoreFactory.getGlobalReader().getSnapshot().getActiveKeysetKeys();
+            Collection<KeysetKey> globalKeysetKeys = keysetKeyStoreFactory.getGlobalReader().getSnapshot().getAllKeysetKeys();
```

Comment:
hey @cody-constine-ttd, I've checked out your branch, run `docker-compose up` and have tried running the application via 
```
mvn clean compile exec:java -Dvertx-config-path=conf/local-config.json -Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory -Dlogback.configurationFile=conf/logback.xml
```
but I am getting compilation errors `cannot find symbol` wherever you have `getAllKeysetKeys`, e.g. here and in EncryptionKeyService.java. Am I doing something wrong? It used to run on a previous commit.

Reply:
try with the latest version of uid2-shared